code/game/machinery/vending.dm

```diff
@@ -1042,6 +1043,7 @@
 					/obj/item/weapon/storage/box/shotgunammo/slug = 900,
 					/obj/item/weapon/storage/box/shotgunammo/buckshot = 900,
 					/obj/item/weapon/tool/knife/tacknife = 600,
+					/obj/item/weapon/storage/box/smoke = 200,
```

Comment:
Add an `s` at the end of smoke. `/obj/item/weapon/storage/box/smoke` does not exist